web/app/src/components/organizms/Sidebar.svelte

```diff
@@ -41,9 +43,11 @@
       <Item href="javascript:void(0)" on:click={() => setActive('reservation')} activated={active === 'reservation'}>
         <Text class="text-base">予約一覧</Text>
       </Item>
-      <Item href="javascript:void(0)" on:click={() => setActive('product')} activated={active === 'product'}>
-        <Text class="text-base">出品一覧</Text>
-      </Item>
+      {#if [USER_ATTRIBUTE.producer, USER_ATTRIBUTE.consumer].includes($profile.attribute)}
```

Comment:
こちらの$profile.attributeの部分で以下のようなエラーが出てしまい、ログインができない状態になってしまっておりました🙏
$profiileのstoreの値が空っぽの状態でログインを試みようとすると起こるエラーだと認識しております。こちら以下のようにオプショナルチェーンを付与して修正するのはいかがでしょうか？
私の動作確認ミスであったら申し訳ございません！！🙇‍♂️
```suggestion
      {#if [USER_ATTRIBUTE.producer, USER_ATTRIBUTE.consumer].includes($profile?.attribute)}
```

> Sidebar.svelte:46 Uncaught (in promise) TypeError: Cannot read properties of null (reading 'attribute')
    at Array.create_default_slot_2 (Sidebar.svelte:46:81)
    at create_slot (index.mjs:69:27)
    at Array.create_default_slot (chunk-TKD4GBKJ.js?v=a9029ff0:74:24)
    at create_slot (index.mjs:69:27)
    at create_fragment12 (Main.svelte:2:21)
    at init (index.mjs:2012:37)
    at new Nav (Nav.svelte:2:21)
    at construct_svelte_component_dev (index.mjs:2213:26)
    at create_fragment (List.svelte:2:18)
    at init (index.mjs:2012:37)

Reply:
@keis8221 
ご指摘ありがとうございます。考慮が漏れておりました。
c62fa20 にて対応しましたのでご確認お願いします。

Reply:
@T-Ishikawa-R 
対応が遅れてしまい申し訳ございません！
LGTMです！